howso/utilities/installation_verification.py

```diff
@@ -576,7 +576,7 @@ def generate_dataframe(*, client: AbstractHowsoClient,
     return df, elapsed_time
 
 
-def check_not_emulated(*, registry: InstallationCheckRegistry):
+def check_not_emulated():
```

Comment:
This one worries me. I checked the usages and it doesn't seem to provide the arg, but unsure.

Reply:
Seems that was required. Putting it back with #noqa ARG001